server/world/viewer.go

```diff
@@ -25,9 +25,10 @@ type Viewer interface {
 	// ViewEntityTeleport views the teleportation of an entity. The entity is immediately moved to a different
 	// target position.
 	ViewEntityTeleport(e Entity, position mgl64.Vec3)
-	// ViewChunk views the chunk passed at a particular position. It is called for every chunk loaded using
-	// the world.Loader.
-	ViewChunk(pos ChunkPos, c *chunk.Chunk, blockNBT map[cube.Pos]Block)
+	// ViewSubChunks views the sub-chunks requested in the offsets passed.
+	ViewSubChunks(center SubChunkPos, offsets [][3]int8)
```

Comment:
I don't think this needs to be added to the Viewer interface. It is only called within the session package and is more of an implementation detail.